### PR TITLE
close #22 View Users List

### LINKED ISSUE #22: View Users List

### PR DESCRIPTION
**Pull request by HaiTranVar**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,9 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "2.4.6"
 
+gem 'jquery-rails'
+gem 'will_paginate'
+gem 'bootstrap-will_paginate'
```
Comment (mai-123) [inline Gemfile]: let replace \'  to \"

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.paginate(page: params[:page])
```
Comment (mai-123) [inline app/controllers/users_controller.rb]: Let remove ()

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <th><%= t ".id" %></th>
+      <th><%= t ".user_name" %></th>
+      <th><%= t ".user_email" %></th>
+      <th><%= t ".action_delete" %></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @users.each do |user| %>
+      <tr>
+        <td><%= link_to user.id, user %></td>
```
Comment (mai-123) [inline app/views/users/index.html.erb]: remove link_to, just show information

File: config/routes.rb [hunk 4]
```diff
@@ -1,9 +1,11 @@
 Rails.application.routes.draw do
+  get 'users/index'
```
Comment (mai-123) [inline config/routes.rb]: remove it

File: app/helpers/users_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (ducminhvu) [inline app/helpers/users_helper.rb]: remove this file

File: app/assets/javascripts/application.js [hunk 6]
```diff
@@ -1,16 +1,2 @@
...
-// compiled file. JavaScript code in this file should be added after the last require_* statement.
-//
-// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
-// about supported directives.
-//
-//= require rails-ujs
-//= require activestorage
-//= require turbolinks
-//= require_tree .
+//= require jquery
```
Comment (ducminhvu) [inline app/assets/javascripts/application.js]: jquery3
https://github.com/rails/jquery-rails

File: app/views/layouts/_header.html.erb [hunk 7]
```diff
@@ -2,13 +2,47 @@
...
+        <li><%= link_to t(".home"), root_path %></li>
         <% if logged_in? %>
-         <li><%= link_to t(".log_out"), logout_path %></li>
+          <% if current_user.is_admin %>
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                <%= current_user.name %><b class="caret"></b>
+              </a>
+              <ul class="dropdown-menu">
+                <li><%= link_to "Profile", "#" %></li>
```
Comment (ducminhvu) [inline app/views/layouts/_header.html.erb]: i18n

**Pull request merged**